### ISSUE TITLE
https connection with certificates in README

### ISSUE THREAD

**Issue by sumdog** [swipely/docker-api#496]
The following Docker command works correctly and pulls version information from the server. 

    docker --tlsverify --tlscacert=docker/ca.crt  --tlscert=docker/docker-client.crt  --tlskey=docker/docker-client.pem  -H=example.com:2376 version

I attempted the same thing using `docker-api`:

     cert_path = 'docker/'

    Docker.url = "https://example.com:2376"
    Docker.options = {
      client_cert: File.join(cert_path, 'docker-client.crt'),
      client_key: File.join(cert_path, 'docker-client.pem'),
      ssl_ca_file: File.join(cert_path, 'ca.pem'),
      scheme: 'https'
    }

    @log.info(Docker.version)

and get the following error:

    .gem/ruby/2.1.0/gems/excon-0.54.0/lib/excon/ssl_socket.rb:116:in `connect_nonblock': SSL_connect returned=1 errno=0 state=error: certificate verify failed (OpenSSL::SSL::SSLError) Unable to verify certificate. This may be an issue with the remote host or with Excon. Excon has certificates bundled, but these can be customized: (Excon::Error::Certificate)

            `Excon.defaults[:ssl_ca_path] = path_to_certs`
            `ENV['SSL_CERT_DIR'] = path_to_certs`
            `Excon.defaults[:ssl_ca_file] = path_to_file`
            `ENV['SSL_CERT_FILE'] = path_to_file'
            `Excon.defaults[:ssl_verify_callback] = callback`
                (see OpenSSL::SSL::SSLContext#verify_callback)
     or:
            `Excon.defaults[:ssl_verify_peer] = false` (less secure).

If I specify the `ssl_verify_peer` to false, I get the correct response: 

    I, [2017-08-10T00:07:09.128287 #17738]  INFO -- : {"Version"=>"1.12.6", "ApiVersion"=>"1.24", "GitCommit"=>"78d1802", "GoVersion"=>"go1.7.4", "Os"=>"linux", "Arch"=>"amd64", "KernelVersion"=>"4.10.0-28-generic", "BuildTime"=>"2017-03-14T09:47:15.250641567+00:00"}

Am I specifying the certificates incorrectly? Is the Readme still valid?

**Comment by andrewwardrobe** [comment]
Did you ever fix this? I'm getting a similar issue

**Comment by SumGR** [comment]
> Did you ever fix this? I'm getting a similar issue

I went back and [checked my code](https://github.com/sumdog/bee2/blob/master/lib/dockerhandler.rb#L65) and it looks like no. I'm still setting `ssl_verify_peer` to false. I tried upgrading to `docker-api` version `1.34.2` (which pulls in `excon 0.64.0`) and I still get the same issue. 

There should be a way to tell `excon` which certification you want to use for verification (since you're going to generate self signed ones for a docker host), but I haven't dug around enough to see how or if it's possible to pass that data from `docker-api` down into `excon`.
